packages/dashboard/src/components/Card/Chart/Chart.types.ts

```diff
@@ -59,6 +59,11 @@ export interface IChartProps {
    */
   data?: IDataPoint[];
 
+  /**
+   * Data to render in the chart.
+   */
+  multipleChartsData?: IDataPoint[][];
```

Comment:
I'm tempted to say we need some renaming here, the singular data variable should be called dataPoints and this one should be called data? @joetheday maybe you can suggest some naming for chart points that contain multiple data arrays vs those that don't

Reply:
As per updated work items, I see that every DataViz will have multiple implementation in future. So why not rename 'multipleChartsData' as 'data' and rename the current 'data' attribute as something else. Also, we need not worry about what we rename the current 
'data' attribute as it will no longer be used in the future, once all the DataViz have multiple implementation.

Reply:
Why don't we just do `data` and have the chart do the deciding what to do? Something like...
```
if data.length > 1
   do this
else
   do that
```

Reply:
For now, I'd recommend @nulikartik's suggestion of renaming the existing `data` attribute since it will eventually be removed. This is a crawl-walk run approach:
Step 1: keep the existing `data` attribute and rename it to, e.g. `singleDataPoints`
Step 2: all charts use a multiple data points implementation
Step 3: remove the `singleDataPoints` attribute

Reply:
Changing name 'multipleChartsData' to 'data'